Odata-docs/odatalib/v7/edm/iedmtargetpath-for-annotations-target.md

```diff
@@ -0,0 +1,463 @@
+        {
+            // Read Annotations
+            string csdl = @"<?xml version=""1.0"" encoding=""UTF-8""?>
+<edmx:Edmx xmlns:edmx=""http://docs.oasis-open.org/odata/ns/edmx"" Version=""4.0"">
+	<edmx:DataServices>
+		<Schema xmlns=""http://docs.oasis-open.org/odata/ns/edm"" Namespace=""NS"">
+			<ComplexType Name=""Address"">
+				<Property Name=""Road"" Type=""Edm.String"" Nullable=""false"" />
+				<NavigationProperty Name=""City"" Type=""NS.City"" Nullable=""false"" />
+			</ComplexType>
+			<ComplexType Name=""DerivedAddress"" BaseType=""NS.Address"">
+				<Property Name=""MyRoad"" Type=""Edm.String"" Nullable=""false"" />
+			</ComplexType>
+			<EntityType Name=""Customer"">
+				<Key>
+					<PropertyRef Name=""Id""/>
+				</Key>
+				<Property Name=""Id"" Type=""Edm.Int32"" Nullable=""false"" />
+				<Property Name=""Name"" Type=""Edm.String"" Nullable=""false"" />
+				<Property Name=""Address"" Type=""NS.Address"" Nullable=""false"" />
+				<NavigationProperty Name=""Cities"" Type=""Collection(NS.City)"" />
+			</EntityType>
+			<EntityType Name=""VipCustomer"" BaseType=""NS.Customer"" />
+			<EntityType Name=""City"">
+				<Property Name=""Name"" Type=""Edm.String"" Nullable=""false"" />
+			</EntityType>
+			<Term Name=""MyTerm"" Type=""Edm.String"" />
+			<EntityContainer Name=""Default"">
+				<EntitySet Name=""Customers"" EntityType=""NS.Customer"" />
+				<Singleton Name=""Me"" Type=""NS.Customer"" />
+			</EntityContainer>
+			<Annotations Target=""NS.Default/Customers/Name"">
+				<Annotation Term=""NS.MyTerm"" String=""Name OutOfLine MyTerm Value"" />
+			</Annotations>
+		</Schema>
+	</edmx:DataServices>
+</edmx:Edmx>"
+            ;
+
+            IEdmModel parsedModel;
+            IEnumerable<EdmError> errors;
+            CsdlReader.TryParse(XElement.Parse(csdl).CreateReader(), out parsedModel, out errors);
+            IEdmTargetPath tgtPath = parsedModel.GetTargetPath("NS.Default/Customers/Name");
+            IEdmTerm fooBarTerm = parsedModel.FindDeclaredTerm("NS.MyTerm");
+            string nameAnnotation = GetAnnotation(parsedModel, tgtPath, fooBarTerm, EdmVocabularyAnnotationSerializationLocation.OutOfLine);
+            Console.WriteLine(nameAnnotation);
+        }
+
+        private static string GetAnnotation(IEdmModel model, IEdmVocabularyAnnotatable target, IEdmTerm term, EdmVocabularyAnnotationSerializationLocation location)
+        {
+            IEdmVocabularyAnnotation annotation = model.FindVocabularyAnnotations<IEdmVocabularyAnnotation>(target, term).FirstOrDefault();
+            if (annotation != null)
+            {
+                IEdmStringConstantExpression stringConstant = annotation.Value as IEdmStringConstantExpression;
+                if (stringConstant != null)
+                {
+                    return stringConstant.Value;
+                }
+            }
+
+            return null;
+        }
+    }
+}
+```
+
+Run the console application.
```

Comment:
If the preceding header is a level 2, you should make this a level 3. It might be confusing to have two "Run the console application" headers in the same document